autoPlay.user.js

```diff
@@ -61,7 +62,7 @@
 	var control = {
 		speedThreshold: 2000,
 		// Stop using offensive abilities shortly before rain/wormhole rounds.
-		rainingSafeRounds: 5,
+		rainingSafeRounds: 9,
```

Comment:
fair enough I'll adjust canUseOffensiveAbility() to +4 to scope it to just office skills.